src/status_im/contexts/wallet/swap/utils.cljs

```diff
@@ -59,23 +59,27 @@
    That happens when token is being selected on the home screen and 
    it basically indicates that no account pre-selection was made."
   [{:keys [wallet account test-networks-enabled? token-symbol]}]
-  (let [networks (-> (get-in wallet [:networks (if test-networks-enabled? :test :prod)])
-                     (network-utils/sorted-networks-with-details))
-        token    (->> account
-                      :tokens
-                      (filter #(= token-symbol (:symbol %)))
-                      first)]
-    (assoc token :networks (network-utils/network-list token networks))))
+  (let [networks         (-> (get-in wallet [:networks (if test-networks-enabled? :test :prod)])
+                             (network-utils/sorted-networks-with-details))
```

Comment:
nit: We are using `->` for only one step:

```clojure
(-> (get-in wallet [])
    (do-something)
```

Better to extract `wallet`, since threading macros of only one step do not have a big benefit:

```clojure
(-> wallet
    (get-in [])
    (do-something)
```
```